chainstate/src/detail/chainstateref/mod.rs

```diff
@@ -143,7 +143,6 @@ impl<'a, S: BlockchainStorageRead, V: TransactionVerificationStrategy> Chainstat
         }
     }
 
-    // TODO this will go when transaction verifier is ready
```

Comment:
I think the comment is valid but can be improved. What Roy wanted to say is that when `tx_verifier` is integrated into mempool we can remove this exposure of utxos (and probably a bunch of other stuff).